app/api-v1/routes/recipe.js

```diff
@@ -5,7 +5,27 @@ module.exports = function (recipeService) {
       res.status(500).json({ message: 'Not Implemented' })
     },
     POST: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      if (!req.body) {
+        return res.status(400).json({ message: 'Invalid request' })
+      }
+
+      const { id, externalId, name, imageAttachmentId, material, alloy, price, requiredCerts, supplier } = req.body
+
+      const recipe = await recipeService.createRecipe({
```

Comment:
why we can not pass req.body. Does it mean that db schema / knex does not do any validation? in mst cases it should remove property and look at the schema for what is required and stuff.. seems like unnecessary deconstruction, but I might be wrong

Reply:
not sure what validation is done by the http layer, but with deconstruction then you would allow anything through to the db? Admittedly we only have the columns we have, I just thought that it would be more specific/strongly defined to do it this way.

Reply:
this may simplify yes. See how the tests go